src/Chapter2.hs

```diff
@@ -858,7 +889,8 @@ and reverses it.
   function, but in this task, you need to implement it manually. No
   cheating!
 -}
-rewind = error "rewind: Not Implemented!"
+rewind :: [Int] -> [Int]
+rewind = foldl (\acc x -> x:acc) []
```

Comment:
Very clever solution! If you want, you can use a standard function `flip` here to go full point-free :smile: 
```suggestion
rewind = foldl (flip (:)) []
```